spec/dummy/db/schema.rb

```diff
@@ -13,9 +13,16 @@
 
 ActiveRecord::Schema.define(version: 20131117183023) do
 
+  create_table "companies", force: true do |t|
+    t.string   "name"
```

Comment:
Put one space between the method name and the first argument.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.